test/UnitTests/MSTestAdapter.UnitTests/Discovery/TestMethodValidatorTests.cs

```diff
@@ -56,6 +56,8 @@ public void IsValidTestMethodShouldReturnFalseForGenericTestMethodDefinitions()
     public void IsValidTestMethodShouldReportWarningsForGenericTestMethodDefinitions()
     {
         SetupTestMethod();
+
+        // _mockMethodInfo.Setup(mi => mi.Module.Assembly.ReflectionOnly).Returns(false);
         _mockMethodInfo.Setup(mi => mi.IsGenericMethodDefinition).Returns(true);
```

Comment:
The code above was not called before most likely, because MOQ cannot mock it, because Assembly is not serializable.